src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -155,28 +163,33 @@ public void populateGridStorage( final IGridStorage storage )
 
 	}
 
+	public <T extends IAEStack<T>, C extends IStorageChannel<T>> IMEInventoryHandler<T> getInventoryHandler( IStorageChannel<T> channel )
+	{
+		return (IMEInventoryHandler<T>) this.storageNetworks.computeIfAbsent( channel, c -> this.buildNetworkStorage( c ) );
+	}
+
+	@Override
+	public <T extends IAEStack<T>, C extends IStorageChannel<T>> IMEMonitor<T> getInventory( IStorageChannel<T> channel )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') C is not used in the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2326)